anchore_engine/services/apiext/api/controllers/images.py

```diff
@@ -1160,11 +1160,16 @@ def analyze_image(account, source, force=False, enable_subscriptions=None, annot
                     image_check = client.get_image(digest)
                     if not image_check:
                         raise Exception('No image found for digest {}'.format(digest))
+                    if not ts:
+                        # Timestamp required for analysis by digest & tag (if none specified,
+                        # default to previous image's timestamp)
+                        ts = image_check.get('created_at', utils.datetime_to_epoch(datetime.datetime.now()))
```

Comment:
time.time() does the same thing, obviating the need for the custom function to convert time formats. But also, be sure that it's using UTC time not the local time on the host.

Reply:
Also see: https://github.com/anchore/anchore-engine/blob/master/anchore_engine/db/entities/common.py#L78

Could probably hoist that into the utils module but that can be in different commit/PR since that impacts a lot of other code. But the above links shows how we set the epochs in the image timestamps.

Reply:
Nice, didn't see the other common method on the first pass. Will use the same method (**anchore_now()**) in this change, then we can overhaul it into utils later.